ex2_hal/adcs/equipment_handler/src/adcs_handler.c

```diff
@@ -3018,25 +3042,38 @@ ADCS_returnState ADCS_set_mtm_config(mtm_config params, uint8_t mtm) {
         return ADCS_INVALID_PARAMETERS;
     }
     xyz16 raw_val_angle, raw_val_offset;
-    float coef = 0.01;
-    raw_val_angle.x = params.mounting_angle.x / coef;
-    raw_val_angle.y = params.mounting_angle.y / coef;
-    raw_val_angle.z = params.mounting_angle.z / coef;
-    memcpy(&command[1], &raw_val_angle, 6);
-    coef = 0.001;
-    raw_val_offset.x = params.channel_offset.x / coef;
-    raw_val_offset.y = params.channel_offset.y / coef;
-    raw_val_offset.z = params.channel_offset.z / coef;
-    memcpy(&command[7], &raw_val_offset, 6);
+    float coef = 100;
+    raw_val_angle.x = params.mounting_angle.x * coef;
+    raw_val_angle.y = params.mounting_angle.y * coef;
+    raw_val_angle.z = params.mounting_angle.z * coef;
+    command[1] = raw_val_angle.x & 0xFF;
+    command[2] = raw_val_angle.x >> 8;
```

Comment:
When doing bit shfits like this it's common to and them with FF after shifting or with FF00 before shifting